denizen_scripts/survival/repo_link/systems/vote_handlers.dsc

```diff
@@ -0,0 +1,151 @@
+      - if <player.flag[daily_key_pending]> > 0:
+        - if <player.flag[daily_key_pending]> > 1:
+          - narrate "<&c>You have <player.flag[daily_key_pending]> <item[daily_vote_key].display>s<&c> waiting for you at <&3>/<&b>recoverkeys."
+        - else:
+          - narrate "<&c>You have a <item[daily_vote_key].display><&c> waiting for you at <&3>/<&b>recoverkeys."
+      - if <player.flag[enderchest_daily_key_offline].add[<player.flag[inventory_daily_key_offline]>]> > 0:
+        - if <player.flag[enderchest_daily_key_offline].add[<player.flag[inventory_daily_key_offline]>]> > 1:
+          - announce "<&c><player.display_name> has recieved <&a><player.flag[enderchest_daily_key_offline].add[<player.flag[inventory_daily_key]>]> <item[daily_vote_key].display>s<&c> for voting while offline!<&nl><&c>Do <&3>/<&b>vote<&c> to recieve yours now!"
+        - else:
+          - announce "<&a><player.display_name> <&c>has just recieved a <item[daily_vote_key].display><&c>! Do <&3>/<&b>vote<&c> to recieve yours now!""
+
+
+vote_handler:
+  type: task
+  debug: false
+  daily_vote_counter:
+    - flag <[voter]> daily_votes_reward counter:++
+    - flag <[voter]> DailyVotesTotal counter:++
+    - wait 1t
+    - if <[voter].flag[daily_votes_reward]> >= <[listed_sites].sub[1]>:
+      - if <[voter].inventory.is_full>:
+        - if <[voter].enderchest.is_full>:
+          - flag <[voter]> daily_key_pending counter:++
+          - flag <[voter]> daily_votes_reward:!
+          - if <[voter].is_online>:
+            - narrate <[voter]> "<&4>Your inventory, and enderchest was full, your <item[daily_vote_key].display> is waiting for you at <&3>/<&b>recoverkeys."
+        - else:
+          - give to:<[voter].enderchest> daily_vote_crate_key
+          - flag <[voter]> daily_votes_reward:!
+          - flag <[voter]> offline_daily_keys counter:++
+          - if <[voter].is_online>:
+            - narrate <[voter]> "<&c>Your inventory was full, your key has been deposited into your enderchest inventory."
+            - announce "<&a><[voter].display_name> <&c>has just recieved a <item[daily_vote_key].display><&c>! Do <&3>/<&b>vote<&c> to recieve yours now!"
+          - else:
+            - flag <[voter]> enderchest_daily_key_offline counter:++
+            - flag <[voter]> offline_daily_keys counter:++
+      - else:
+        - give <[voter]> daily_vote_crate_key
+        - flag <[voter]> daily_votes_reward:!
+        - if <[voter].is_online>:
+          - announce "<&a><[voter].display_name> <&c>has just recieved a <item[daily_vote_key].display><&c>! Do <&3>/<&b>vote<&c> to recieve yours now!"
+        - else:
+          - flag <[voter]> inventory_daily_key_offline counter:++
+          - flag <[voter]> offline_daily_keys counter:++
+
+
+  weekly_vote_counter:
+    - flag <[voter> weekly_votes_reward counter:++ duration:7d
+    - wait 1t
+    - if <[voter].flag[weekly_votes_reward].sub[1].mul[7]> >= <[listed_sites]>:
```

Comment:
this is likely meant to be checking the size of the listed site's list;
```suggestion
    - if <[voter].flag[weekly_votes_reward].sub[1].mul[7]> >= <[listed_sites].size>:
```

Reply:
see previous comment regarding element vs list